cdk/assets/scheduled_task_exec/index.js

```diff
@@ -1,38 +1,41 @@
-const aws = require('aws-sdk')
+const aws = require('aws-sdk');
 
-const ECS_CLUSTER = process.env.ECS_CLUSTER;
-const ECS_SERVICE = process.env.ECS_SERVICE;
-const ECS_TASK_DEFINITION = process.env.ECS_TASK_DEFINITION;
+const { ECS_CLUSTER } = process.env;
+const { ECS_SERVICE } = process.env;
+const { ECS_TASK_DEFINITION } = process.env;
 
 const ecs = new aws.ECS();
 
-exports.handler = async (event, context) => {
+exports.handler = async (event) => {
   const { execCommand = 'python manage.py check' } = event;
 
   const serviceResp = await ecs
     .describeServices({
       cluster: ECS_CLUSTER,
       services: [ECS_SERVICE],
-    }).promise();
+    })
+    .promise();
   const service = serviceResp.services[0];
 
   const { networkConfiguration } = service;
 
-  const execResp = await ecs.runTask({
-    cluster: ECS_CLUSTER,
-    taskDefinition: ECS_TASK_DEFINITION,
-    networkConfiguration,
-    launchType: 'FARGATE',
-    platformVersion: '1.3.0',
-    overrides: {
-      containerOverrides: [
-        {
-          name: 'AppOnlyContainer',
-          command: ['/bin/sh', '-c', execCommand],
-        }
-      ],
-    },
-  }).promise();
+  const execResp = await ecs
+    .runTask({
+      cluster: ECS_CLUSTER,
+      taskDefinition: ECS_TASK_DEFINITION,
+      networkConfiguration,
+      launchType: 'FARGATE',
+      platformVersion: '1.4.0',
```

Comment:
this got missed when we moved to v1.4 a while badk

Reply:
Lines like these are a little hard for me to read...maybe formatting like this:

```
const execResp = await ecs.runTask({
  ...
});
```

or 

```
const execResp = await (
  ecs.runTask({
      ...
  })
);
```

Reply:
actually i did and then prettier changed it back. I'm going to just leave it the way prettier dictates rather than track down whatever formatting rule(s) that might involve.